src/AddSupplier.tsx

```diff
@@ -0,0 +1,24 @@
+import { VStack, StackDivider, Box, Flex } from '@chakra-ui/react';
+import AddSupplierTittle from './Supplier/AddSupplierTittle';
+import AddSupplierButtons from './Supplier/AddSupplierButtons';
+import AddSupplierData from './Supplier/AddSupplierData/AddSupplierData';
+
+function Form() {
```

Comment:
It's a good practice to keep components in separate folder, not in root `src` folder.
I've created `components` folder and inside it `table` folder where I store all components connected with table.